AncientReligions/gfx/flags/d_gwydion.tga

```diff
@@ -0,0 +1,731 @@
+
```

Comment:
That's because that's exactly what it is. No idea how it got in your flags, though. Sometimes github decides to mess up rather spectacularly.